pdf.go

```diff
@@ -17,6 +115,11 @@ func ConvertPDF(r io.Reader) (string, map[string]string, error) {
 	}
 	defer f.Done()
 
+	// Verify if pdf has images or is pdf only-text
+	if PDFHasImage(f.Name()) {
```

Comment:
Looks like you missed a comment from an earlier diff:

Does this mean that if a PDF has an embedded image then it will ignore completely the text of the document?